src/redux/features/signer.ts

```diff
@@ -0,0 +1,70 @@
+import { StdFee } from '@cosmjs/launchpad';
```

Comment:
as I see this file almost not being used, it is needed? lets discuss

Reply:
confirmation is still in use